webapi/src/main/scala/org/knora/webapi/responders/admin/RestProjectsService.scala

```diff
@@ -0,0 +1,45 @@
+package org.knora.webapi.responders.admin
+import zio.Task
+import zio.URLayer
+import zio.ZLayer
+
+import org.knora.webapi.IRI
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectGetRequestADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectGetResponseADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectIdentifierADM
+import org.knora.webapi.responders.ActorToZioBridge
+
+final case class RestProjectsService(bridge: ActorToZioBridge) {
+
+  /**
+   * Finds the project by its [[ProjectIdentifierADM]] and returns the information as a [[ProjectGetResponseADM]].
+   *
+   * @param projectIri           an [[IRI]] identifying the project
+   * @return
+   *     '''success''': information about the project as a [[ProjectGetResponseADM]]
+   *
+   *     '''error''': [[dsp.errors.NotFoundException]] when no project for the given IRI can be found
+   *                  [[dsp.errors.ValidationException]] if the given `projectIri` is invalid
+   */
+  def getSingleProjectADMRequest(projectIri: IRI): Task[ProjectGetResponseADM] =
+    ProjectIdentifierADM.IriIdentifier
+      .fromString(projectIri)
```

Comment:
What's the reason for using `IRI` here, instead of `ProjectIdentifierADM`? Otherwise it would work for all other identifiers too, right?

Reply:
This is overloaded and exposes the method to the route.